pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type accessTokenOptions struct {
+	namespace      string
+	serviceAccount string
+	roleBinding    string
+}
+
+var accessTokenOpts *accessTokenOptions
+
+var accessTokenExamples = strings.Trim(`
+Create an access token in a leader cluster for one or more member clusters, if the Service Account or RoleBinding does not exit, antctl will create one
```

Comment:
, if -> . If

Reply:
You used "ServiceAccount" in other commands. Again, could you check and use consistent formatting?

Reply:
done. Rename another two "Service Account" to "ServiceAccount" .